tests/lib/snaps/test-snapd-ssh-keys/bin/check

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+head -n1 /etc/ssh/ssh_config
```

Comment:
This is my personal preference but I think it can be argued about: can you please just expose `sh` from the test snap and put this login into the spread job itself. This way the casual reader will just need to see the `task.yaml` as the test snap won't carry any logic with it.